.env.sample

```diff
@@ -0,0 +1,3 @@
+CS_ENV='environment'
+CS_API_KEY='your api key'
+CS_TOKEN='access token'
```

Comment:
Could you please:

a) rename `CS_ENV` to `CS_DEVELOPMENT_ENV` and add `CS_PRODUCTION_ENV` variable.
delelopment.js config with `CS_DEVELOPMENT_ENV` variable will be used for `npm run dev`, when production.js should use `CS_PRODUCTION_ENV` and will be invoked for `npm run start`

b) rename change `CS_TOKEN='access token'` to `CS_DELIVERY_TOKEN`='delivery token'` because
access token and delivery token are two different tokens in CS terminology. You can find more information here - https://www.contentstack.com/docs/developers/create-tokens/types-of-tokens/

Reply:
Thanks for the links, fixed.